add warning about using the correct user to usage.md (#211)
thelounge/thelounge.github.io: _docs/usage.md

thelounge version
```

## Using the correct system user

Note that all commands **must** be executed as the same user The Lounge will be run as.

### Unix like systems:

> If you installed The Lounge via the package manager of your distro and plan to run it as a system service, the user is called "thelounge". So every command needs to be executed as `sudo -u thelounge thelounge <command>`, where `<command>` should be substituted with a subcommand like `start` or `add`

Reviewer: I'd remove the sub heading.

```suggestion
If you installed The Lounge via the package manager on a unix-like system and plan to run it as a system service, the user is called `thelounge`. Every command needs to be executed as `sudo -u thelounge thelounge <command>`, where `<command>` should be substituted with a sub command like `add` or `install`.
```

Author: done
